Dockerfile

```diff
@@ -11,6 +11,7 @@ LABEL maintainer="Paulus Schoutsen <Paulus@PaulusSchoutsen.nl>"
 #ENV INSTALL_FFMPEG no
 #ENV INSTALL_LIBCEC no
 #ENV INSTALL_PHANTOMJS no
+#ENV INSTALL_CHROME no
```

Comment:
This line should be in [`Dockerfile.dev`](https://github.com/home-assistant/home-assistant/blob/dev/virtualization/Docker/Dockerfile.dev) too